examples/encoding/plot_encoding_model.py

```diff
@@ -0,0 +1,145 @@
+"""
+========================================
+Regression on continuous and event-based data)
+========================================
+Neat description. Oooo so cool.
+
+rERPs are described in:
+
+  Smith, N. J., & Kutas, M. (2015). Regression-based estimation of ERP
+  waveforms: II. Non-linear effects, overlap correction, and practical
+  considerations. Psychophysiology, 52(2), 169-189.
+
+Estimation of receptive fields and modeling with continuous inputs
+is described in:
+
+  Theunissen, F. E. et al. Estimating spatio-temporal receptive
+  fields of auditory and visual neurons from their responses to
+  natural stimuli. Network 12, 289-316 (2001).
+
+  Willmore, B. & Smyth, D. Methods for first-order kernel
+  estimation: simple-cell receptive fields from responses to
+  natural scenes. Network 14, 553-77 (2003).
```

Comment:
Or:
Lalor, E. C., & Foxe, J. J. (2010). Neural responses to uninterrupted natural speech can be extracted with precise temporal resolution. European journal of neuroscience, 31(1), 189-193.